index.html

```diff
@@ -151,7 +151,7 @@ <h1>DRAFT</h1>
         </div>
 
         <p class="mission">
-            The <strong>mission</strong> of the <a href="https://www.w3.org/2021/lds-wg/" class=todo>Linked Data Signatures Working Group</a> is to define a standard for uniquely identifying and securing <a href='https://www.w3.org/standards/semanticweb/data'>Linked Data</a> used in use cases such as <a href="https://www.w3.org/TR/vc-data-model">Verifiable Credentials</a>. The work will include defining <a href="explainer.html#canonicalization"> RDF Dataset Canonicalization</a> algorithms as well as algorithms and vocabularies for encoding digital proofs, such as <a href='https://en.wikipedia.org/wiki/Digital_signature'>digital signatures</a>, and with that secure information expressed in serializations such as <a href="https://www.w3.org/TR/json-ld/">JSON-LD</a>, <a href="https://www.w3.org/TR/trig/">TriG</a>, and <a href="https://www.w3.org/TR/n-quads/">N-Quads</a>.
+            The <strong>mission</strong> of the <a href="https://www.w3.org/2021/lds-wg/" class=todo>Linked Data Signatures Working Group</a> is to define a standard for uniquely identifying and securing <a href='https://www.w3.org/standards/semanticweb/data'>Linked Data</a> used in use cases such as <a href="https://www.w3.org/TR/vc-data-model">Verifiable Credentials</a>. The work will include defining <a href="explainer.html#canonicalization"> RDF Dataset Canonicalization</a> algorithms as well as algorithms and vocabularies for encoding digital proofs, such as <a href='https://en.wikipedia.org/wiki/Digital_signature'>digital signatures</a>, and with that secure information expressed using <a href="https://www.w3.org/TR/rdf11-concepts/#dfn-concrete-rdf-syntax">concrete RDF syntaxes</a>.
         </p>
```

Comment:
I agree with using the term "concrete RDF syntaxes", but I would still prefer to leave the "such as..." part. This is important for those readers who are not 100% familiar with all these concepts.

Reply:
There is a link to the definition for the reader of a W3C WG charter.. just as there is a link to the definitions/descriptions of every other significant term that's used in the paragraph.

Reply:
And the link should stay. The rest is "marketing", but may be an important one.

Reply:
I'm curious to know about those readers that are familiar with "TriG" and "N-Quads" but not *100% familiar* with "concrete RDF syntax" :)

Alternatively, use "such as [..]":

* "Turtle, RDFa, JSON-LD, and TriG" -- the example list from concrete RDF syntaxes
* "RDFa, JSON-LD" -- general Web at large... this one would be the "marketing" bingo.
* "RDF/XML.." -- for the oldskoolers
* ..

Reply:
> I'm curious to know about those readers that are familiar with "TriG" and "N-Quads" but not 100% familiar with "concrete RDF syntax" :)

To be very honest with you: I have also forgotten that the specification used this term. Hence my usage of the term "serialization" instead :-)

Reply:
+1 to @csarven 's rewording, but also +1 to keep a list of examples. 
Note that, strictly speaking, Turtle, RDFa and RDF/XML are not appropriate examples, because we are talking about datasets here, not graphs.

Reply:
@pchampin  Is it that JSON-LD, TriG, N-Quads syntaxes have native support for graph naming whereas Turtle, RDFa, RDF/XML don't?

The RDF Dataset Canonicalization, Draft CG Report ( https://json-ld.github.io/rdf-dataset-canonicalization/spec/index.html#dfn-rdf-dataset ) defines:

>input dataset
    The abstract RDF dataset that is provided as input to the algorithm.

If canonicalization is happening on RDF datasets, does the encoding of the source data matter? Is the input dataset intended to be in N-Quads (irrespective of prior steps eg. JSON-LD -> N-Quads, Turtle -> N-Quads)?

Reply:
> @pchampin Is it that JSON-LD, TriG, N-Quads syntaxes have native support for graph naming whereas Turtle, RDFa, RDF/XML don't?

yes, that's roughly what I meant (more precisely: JSON-LD, TRiG and N-Quads encode datasets, while Turtle, RDFa and RDF/XML encode graphs).

> If canonicalization is happening on RDF datasets, does the encoding of the source data matter?

no, as long as that encoding encodes a dataset. But with a very strict reading of the definitions in RDF11-CONCEPTS, graphs and datasets are two different beasts (a dataset is a *collection* of graphs).

Reply:
I am not sure where this discussion is going now. @csarven, are you _opposed_ to keeping the list in the charter text (ie, the change would be to use the correct term per the RDF semantics)? Because if not, let us do the changes and merge this...

Reply:
I'm a bit confused. Can you give examples (in code) of valid *input datasets*? I'm trying to understand if "input dataset MUST be in N-Quads.."

If so, what's the point in mentioning anything besides N-Quads (or possibly other RDF syntaxes that the algorithm can work with)? Wouldn't anything prior to the input be out of scope?

Surely, JSON-LD, TriG, ... Turtle, RDFa, RDF/XML + graph name (the 'why' IRI) can be serialized as N-Quads as per requirement.

Reply:
@iherman Where this discussion is hopefully a clarification on the whole paragraph. There are two things being mentioned that could be clarified (IMO):

* RDF syntaxes of interest/relevant for the charter as a whole.
* RDF syntaxes that are required/limited to the RDF Dataset Canonicalization algorithm.

FWIW, I'm fine with "concrete RDF syntaxes such as x, y, z" if that somehow captures both points above.. just trying to understand the reasoning behind x, y, z.

Reply:
I realize I contributed to make the discussion confusing, because the text we are discussing is actually not about the input of the canonicalization algorithm, so the graph/dataset distinction is not so much important here. Sorry about that.

> Can you give examples (in code) of valid input datasets? 

As you pointed out, the canonicalization algorithm expects as input an abstract dataset, so writing it down as code is, by definition, tricky... And no, the algorithm should not care about how it is encoded (although of course, implementations will have to parse some encoding).

But back to the question at hand: there are two changes in your PR
* rewording "serialization" into "concrete syntax", which I think we are all ok with;
* removing the list of examples of such concrete syntaxes.

My proposal was to keep your rewording, but nonetheless keep a list of examples, to illustrate the notion of concrete syntax ("such as", I think, makes it explicit enough that this is not meant to be an exhaustive list).

Which examples we keep in the list is, IMO, not important. Since the rest of the text focuses on datasets rather than graphs, those seem the most relevant, though.

Reply:
> My proposal was to keep your rewording, but nonetheless keep a list of examples, to illustrate the notion of concrete syntax ("such as", I think, makes it explicit enough that this is not meant to be an exhaustive list).

+1 to this, FWIW. Remember that a non-trivial chunk of the communities using these technologies (such as the CCG and people that use JSON-LD) have no idea what a "concrete RDF Syntax" is... just as I assume that much of the RDF community has no idea what a "birationally equivalent elliptic curve" is, but they may have heard of "Ed25519" before. Concrete examples help when referring to abstract concepts.